propagation_ot.go

```diff
@@ -41,8 +41,8 @@ func (p *textMapPropagator) Inject(
 	if !ok {
 		return opentracing.ErrInvalidCarrier
 	}
-	carrier.Set(fieldNameTraceID, strconv.FormatInt(sc.raw.TraceID, 16))
-	carrier.Set(fieldNameSpanID, strconv.FormatInt(sc.raw.SpanID, 16))
+	carrier.Set(fieldNameTraceID, strconv.FormatUint(sc.raw.TraceID, 16))
+	carrier.Set(fieldNameSpanID, strconv.FormatUint(sc.raw.SpanID, 16))
```

Comment:
the propagated span id ends up becoming the parent id for the new span.

https://github.com/opentracing/basictracer-go/blob/bg/make_id_uint64/propagation_ot.go#L112